src/main/groovy/edu/oregonstate/mist/api/Error.groovy

```diff
@@ -113,6 +110,13 @@ class Error {
     }
 }
 
+/**
+ * Object that contains an "about" link member for detailed error information
+ */
+class LinkObject {
```

Comment:
Relevant link for error must be in a links object, in the about member
http://jsonapi.org/format/#error-objects